docs/usage/coodinator.md

```diff
@@ -0,0 +1,159 @@
+# Coordinator
+
+The meta cni-plugin: `coordinator` provides features such as route tuning between pods with multiple NICs, IP address conflict detection, gateway 
+reachability detection, and fixed the Mac address of the pod,etc. Coordinator can work in underlay and overlay mode. 
+
+- In underlay mode, `coordinator` create a pair of veth device,The one pair is placed to the host, The another one is placed in the net namespace of the pod (called veth0). 
+this pair of veth devices helps pods forward east-west traffic from the cluster.
+- In overlay mode, instead of creating a pair of veth device, `coordiantor` forwards east-west traffic to the cluster with the help of the pod's eth0 NIC (often created by some overlay CNI, such as calico or cilium).
```

Comment:
```suggestion
- In overlay mode, rather than creating a pair of veth devices, the `coordinator` forwards east-west traffic to
  the cluster using the pod's eth0 NIC, typically created by an overlay CNI like Calico or Cilium.
```